### DISCUSSION TITLE
Formatter for instance JS files and JS search

### DISCUSSION THREAD

**Discussion by WarDog1982** [arnoudkooi/SN-Utils#541] [Ideas]
Hi,
For those of us that dig into the bowels of SNow, providing a JS formatter and search function for searching the files within the https://#instance#.service-now.com/scripts/ folder would be handy.

Use case is to understand what hidden scripts and functions do, helps when working with support to guide them to issues and also simply to understand more about the platform.

Cheers,
WD

**Comment by arnoudkooi** [comment]
I'm not sure how to get all script files, I think the browser developer tools is your friend here?
